src/utils/github.ts

```diff
@@ -18,15 +18,18 @@ export type Example = {
 
 export function githubRequest(url: string) {
 	const headers: Headers = new Headers({
-		Accept: 'application/vnd.github.v3+json',
-	});
-	if (typeof import.meta.env.PUBLIC_VITE_GITHUB_TOKEN !== 'string') {
+		Accept: "application/vnd.github.v3+json",
+	})
+	if (typeof import.meta.env.GITHUB_TOKEN !== "string") {
```

Comment:
we had a mix of a couple variables here, simplified to a single server-only variable